src/common.rs

```diff
@@ -1303,7 +1309,9 @@ fn path_directories() -> Vec<std::path::PathBuf> {
 pub fn get_delegated_validator_list_from_mainnet(
     network_connection: &linked_hash_map::LinkedHashMap<String, crate::config::NetworkConfig>,
 ) -> color_eyre::eyre::Result<std::collections::BTreeSet<near_primitives::types::AccountId>> {
-    let network_config = network_connection.get("mainnet").expect("Internal error!");
+    let network_config = network_connection
+        .get("mainnet")
+        .expect("There is no 'mainnet' network in your configuration.");
```

Comment:
Let's use `wrap_err()?` instead of expect

Reply:
Done